clone/logdb/database.go

```diff
@@ -56,10 +56,10 @@ func NewDatabaseDirect(db *sql.DB) (*Database, error) {
 }
 
 func (d *Database) init() error {
-	if _, err := d.db.Exec("CREATE TABLE IF NOT EXISTS leaves (id INTEGER PRIMARY KEY, data BLOB)"); err != nil {
+	if _, err := d.db.Exec("CREATE TABLE IF NOT EXISTS leaves (id BIGINT UNSIGNED PRIMARY KEY, data BLOB)"); err != nil {
```

Comment:
Should we try `ALTER TABLE` if the tables do exist (or, perhaps better, print a message with the commands?)

Reply:
Without having a way to query the schema to see what the current column type is defined to be, this would turn into log spam even for users that have already run the commands. Adding code the query the schema is a bit much and is likely to make this more brittle and less portable between MySQL and sqlite (https://stackoverflow.com/questions/1215368/how-to-get-the-mysql-table-columns-data-type).

If we're going to do schema migration properly, we should do it properly and use github.com/golang-migrate/migrate.